_posts/zh/newsletters/2020-01-22-newsletter.md

```diff
@@ -55,6 +55,6 @@ lang: zh
 [whatsat]: https://github.com/joostjager/whatsat
 [news72 sphinx]: /zh/newsletters/2019/11/13/#possible-privacy-leak-in-the-ln-onion-format
 [river twitter thread]: https://twitter.com/philipglazman/status/1216849483184476165
-[wasabi rbf notification]: /en/compatibility/wasabi/
```

Comment:
I think all of these should work when it’s deployed, because there are redirects from 

    /en/compatibility/wasabi/ ⇒ /en/matrix/
    /en/compatibility/ ⇒ /en/matrix/
    
in en/matrix.md line 6–8.

Reply:
Just to be clear:  no action is expected here.  Correct?__

Reply:
Yes, I originally had caught a couple instances in the first commit that had not been fixed yet, then later realized that there was a redirect, and finally discovered that another commit later by @bitschmidty replaced them.

As far as I can tell, everything should be fine.